pkg/controller/master/supportbundle/register.go

```diff
@@ -13,26 +15,32 @@ const (
 func Register(ctx context.Context, management *config.Management, options config.Options) error {
 	sbs := management.HarvesterFactory.Harvesterhci().V1beta1().SupportBundle()
 	settings := management.HarvesterFactory.Harvesterhci().V1beta1().Setting()
-	nodes := management.CoreFactory.Core().V1().Node()
-	nodesCache := nodes.Cache()
+	nodeCache := management.CoreFactory.Core().V1().Node().Cache()
+	podCache := management.CoreFactory.Core().V1().Pod().Cache()
 	deployments := management.AppsFactory.Apps().V1().Deployment()
 	daemonsets := management.AppsFactory.Apps().V1().DaemonSet()
 	services := management.CoreFactory.Core().V1().Service()
 
-	controller := &Handler{
-		supportBundles: sbs,
-		settingCache:   settings.Cache(),
-		nodeCache:      nodesCache,
-		deployments:    deployments,
-		daemonSets:     daemonsets,
-		services:       services,
+	handler := &Handler{
+		supportBundles:          sbs,
+		supportBundleController: sbs,
+		settingCache:            settings.Cache(),
```

Comment:
nit: use a settingCache variable so that it looks consistent with the rest

Reply:
Fixed in https://github.com/harvester/harvester/pull/844/commits/4d0240934fdb0f340e98b0d5afc70d13e9a1dd46